services/pull/merge.go

```diff
@@ -871,7 +871,7 @@ func MergedManually(pr *issues_model.PullRequest, doer *user_model.User, baseGit
 		return err
 	}
 
-	notification.NotifyMergePullRequest(pr, doer)
+	notification.NotifyMergePullRequest(db.DefaultContext, doer, pr)
```

Comment:
If we already have access to a more specialized context than `db.DefaultContext` (`baseGitRepo.Ctx`), shouldn't we use that instead?
```suggestion
	notification.NotifyMergePullRequest(baseGitRepo.Ctx, doer, pr)
```